modules/virt-about-node-maintenance.adoc

```diff
@@ -7,6 +7,11 @@
 
 Nodes can be placed into maintenance mode using the `oc adm` utility, or using `NodeMaintenance` custom resources (CRs).
 
+[NOTE]
+====
+As of version 4.11, `node-maintenance-operator` (NMO) is no longer shipped with {VirtProductName}. It is now available to deploy as a standalone operator through the Operator Lifecycle Manager (OLM) in the OperatorHub.
```

Comment:
Suggestion to rephrase the second sentence as something like this:
````
It is now available to deploy as a standalone Operator from the *OperatorHub* in the {product-title} web console.
````

Reply:
...or from CLI, by following https://docs.openshift.com/container-platform/4.10/nodes/nodes/eco-node-maintenance-operator.html#installing-maintenance-operator-using-cli_node-maintenance-operator